### PR TITLE
add Mike Prorock to editor list

### PR DESCRIPTION
**Pull request by mprorock**
@decentralgabe should we also remove Orie on this PR or do you want that seperate?

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/pull/232.html" title="Last updated on Dec 15, 2023, 5:26 PM UTC (e27f766)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/232/7544c07...e27f766.html" title="Last updated on Dec 15, 2023, 5:26 PM UTC (e27f766)">Diff</a>

### CODE REVIEW

**Review by iherman — APPROVED** [review]
I presume Orie should also be removed from the CODEOWNERS list, and Mike should be added...

File: index.html [hunk 1]
```diff
@@ -34,6 +34,12 @@
              company: "Block",
              companyURL: "https://www.tbd.website",
              w3cid: 116851
+           }, {
+             name: 'Mike Prorock',
+             url: 'https://github.com/mprorock',
+             company: 'mesur.io',
+             companyURL: 'https://mesur.io/',
+             w3cid: 109171
```
Comment (iherman) [inline index.html:42]: Is this correct? Our database refers to 130636, see https://www.w3.org/users/130636/

**Review by iherman — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (mprorock) [reply index.html:42]: good catch, thanks

Comment (mprorock) [inline index.html:42]:
```suggestion
             w3cid: 130636
```

**Comment by mprorock** [comment]
> I presume Orie should also be removed from the CODEOWNERS list, and Mike should be added...

resolved in [e27f766](https://github.com/w3c/vc-json-schema/pull/232/commits/e27f7668da1d90445dc632e1d72057fc0b5cba07)

**Review by decentralgabe — APPROVED** [review]

**Review by iherman — APPROVED** [review]

**Comment by decentralgabe** [comment]
@TallTed this is the case https://github.com/w3c/vc-json-schema/blob/main/index.html#L57

**Pull request merged**